ui/raidboss/data/04-sb/raid/o3s.txt

```diff
@@ -1,176 +1,136 @@
-#hideall "Place Token (Ninjas/Giant)"
-#hideall "Place Dark Token"
-
-#########TANKS CAN HIDE THESE
-#hideall "Haste"
-#hideall "The Playing Field"
-#hideall "The Game"
-#hideall "Place Token (Apanda)"
-#hideall "Place Token (Ninjas/Giant)"
-#hideall "Place Dark Token"
-
-##############################################################
-################## Windows 8 & 10 Voices ##################
-########REMOVE THE HASTAG to select a voice
-#define speaker "voice" "Microsoft Zira Desktop" 0 100
-#define speaker "voice" "Microsoft David Desktop" 0 100
-
-#################### Windows 7 Voices ####################
-########REMOVE THE HASTAG to select a voice
-#define speaker "voice" "Microsoft Anna" 0 100
-
-################ CUSTOMIZE Call Outs #####################
-########REMOVE THE HASHTAG to enable each call out
-#alertall "Dimensional Wave" before 3 speak "voice" "AoE"
-#alertall "Critical Hit" before 3 speak "voice" "Tank Buster"
-#alertall "Queen's Waltz (Clock)" before 3 speak "voice" "Surround Boss"
-#alertall "Queen's Waltz (Crystals)" before 5 speak "voice" "Go Blue Tile"
-#alertall "Queen's Waltz (Books)" before 5 speak "voice" "Individual Tiles"
-#alertall "Oink" before 3 speak "voice" "stack"
-#alertall "Ribbit" before 1 speak "voice" "ribbit cone"
-#alertall "Squelch" before 3 speak "voice" "look away"
-#alertall "Mindjack" before 0 speak "voice" "look at your debuff"
-#alertall "Magic Hammer" before 5 speak "voice" "stack on individual tiles"
-#alertall "Tanks Morph" before 3 speak "voice" "tanks morph"
-#alertall "Healers Morph" before 3 speak "voice" "healers morph"
-#alertall "DPS Morph" before 3 speak "voice" "DPS morph"
-#alertall "Place Token" before 3 speak "voice" "add spawn"
-
-##############################################################
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
 
-########Phase 1
-0	"Start" sync /:Engage!/ window 0,1
-10	"Critical Hit"	sync /:Halicarnassus:22EB:/ window 10,10	#Tank buster
-19	"Spellblade Holy"	duration 6 #4 players spread/stack
-37	"Queen's Waltz (Clock)"	#Spread around boss
-41	"Haste"
-44	"Random Elemental" #Fire, Blizz, or Thunder
-50	"Dimensional Wave"	#AoE
-59	"The Playing Field"
-65	"Oink"	#Stack with another player
-72	"The Game"	#Stand on role tiles
+# -ii 22EA 22FE 2309 230C 230F 2315
+
+0 "Start" sync /Engage!/ window 0,1
+1.7 "--sync--" sync /:Halicarnassus:22EA:/ window 1.7,2
+10.7 "Critical Hit" sync /:Halicarnassus:22EB:/ window 10.7,5
+19.7 "Spellblade Holy" sync /:Halicarnassus:22EF:/
+25.4 "Pole Shift" # sync /:Halicarnassus:22F[23]:/
+28.4 "Holy Blur/Holy Edge" sync /:Halicarnassus:22F[01]:/
+37.4 "the Queen's Waltz (Clock)" sync /:Halicarnassus:2306:/ window 37.4,5
+37.9 "Sword Dance" sync /:Halicarnassus:2307:/
+41.4 "Haste" sync /:Halicarnassus:22F4:/
+44.8 "Spellblade Thunder III" sync /:Halicarnassus:22EE:/
+50.3 "Dimensional Wave" sync /:Halicarnassus:22F6:/
+59.3 "the Playing Field" sync /:Halicarnassus:2300:/
```

Comment:
the => The, here and elsewhere, if you don't mind?

Reply:
Blast, there's always one set that gets overlooked.